### ISSUE TITLE
Links should use an anchor tag instead of a button

### ISSUE THREAD

**Issue by TimoBbz** [DSD-DBS/capella-collab-manager#121]
In order for the links to have the link features provided by the browser and for an accessibility matter, the tags containing a routerLink attribute should be anchors (<a>) instead of button.
It will allow for exemple to have the feature “Open in a new” when the link is ctrl-clicked.
  
```html
<a mat-raised-button color="primary" routerLink="/projects">
  Projects
</a>
```
should be privilegied over
```html
<button mat-raised-button color="primary" routerLink="/projects">
  Projects
</button>
```

For more informations : https://www.w3.org/WAI/WCAG21/Techniques/html/H91
